cpp/src/parquet/encoding.cc

```diff
@@ -3200,6 +3311,9 @@ class DeltaByteArrayDecoder : public DecoderImpl,
   std::shared_ptr<ResizableBuffer> buffered_data_;
 };
 
+using DeltaByteArrayDecoder = DeltaByteArrayDecoderBase<ByteArrayType>;
+using DeltaLargeByteArrayDecoder = DeltaByteArrayDecoderBase<LargeByteArrayType>;
```

Comment:
ditto

Reply:
I have added a test that validates a regular ByteArray (read non large) can be read even with `use_large_binary_variants`. Is that the test you want? Or you want a test that validates an overflow string can be read using this encoding? If so, is it actually possible? I might be wrong, but I do recall a previous conversation where it was established it will fallback plain encoding due to the string size

Reply:
Thanks for adding the test! I do see that you have extended `TEST(TestArrowReadDeltaEncoding, DeltaByteArray)` to read into large binary. Could you also add a similar one for `TEST(TestArrowReadDeltaEncoding, DeltaBinaryPacked)`? I don't think the overflow case is needed. Just want to make sure all four binary-encodings are covered.